code/lib/channels/src/websocket/index.ts

```diff
@@ -98,7 +111,7 @@ export function createChannel({
     channelUrl = `${protocol}://${hostname}:${port}/storybook-server-channel`;
   }
 
-  const transport = new WebsocketTransport({ url: channelUrl, onError });
+  const transport = new WebsocketTransport({ page: 'preview', url: channelUrl, onError });
```

Comment:
Wasn't sure what to do here. Given this API is deprecated, it doesn't seem like there's much point in adding a `page` argument to it. I'm guessing that most calls will be in the preview but that could be totally wrong.

Reply:
Well if we want to add it here, we should also add it as a argument/option for `createChannel` in this file.

Reply:
@ndelangen I guess there's no reason not to, but my assumption is anyone who would actually be updating their usage would just go ahead and change to the non-deprecated API.